### ISSUE TITLE
Provide styles to normalize drop caps

### ISSUE THREAD

**Issue by JayPanoz** [readium/readium-css#41]
This is part of the accessibility baseline, as defined by issue #18 

We currently normalize everything but drop caps when the a11y flag is set, removing italics, superscripts, etc. 

Drop caps are quite popular in ebooks (for some unknown reason 😛) and their implementation can vary greatly so we need to find some reliable way to normalize them e.g. if using `p:first-letter`, it won’t override `span.dropcap` so we need to go the extra mile, it’s impossible to override `svg` and `img`, etc. 

This normalization is bound to fail with at least some implementations.
